html/index.html

```diff
@@ -208,7 +208,7 @@ <h3 class="text-themecolor">TAB 1</h3>
             <!-- ============================================================== -->
             <!-- footer -->
             <!-- ============================================================== -->
-            <footer class="footer"> © 2020 JECRC-IT <a href="https://www.wrappixel.com/">wrappixel.com</a> </footer>
+            <footer class="footer"> © 2020 JECRC-IT </footer>
```

Comment:
Please do this as is it uniform in all the pages
```suggestion
            <footer class="footer"> © 2020 JECRC-IT by <a href="">contributors</a></footer>
```